src/engine.cpp

```diff
@@ -155,7 +155,7 @@ void Engine_Start(int argc, char **argv)
     // Primary initialization.
     Engine_Init_Pre();
 
-    Engine_LoadConfig(config_name ? config_name : "config.lua");
+    Engine_LoadConfig(config_name);
```

Comment:
Approved, also fixes bug where config_name is always true (it shouldn't be treated as pointer) maybe it is previously treated as pointer due to config_name previously being of type char*

Reply:
Yes, it previously was a pointer. I forgot to change this line in my previous commit c1bd68b76b4a7617e0c963fe76b87ad76445d1ad.